default.nix

```
@@ -21,7 +21,7 @@ pkgs.python311.pkgs.buildPythonApplication {
   makeWrapperArgs = [
     "--prefix PATH"
     ":"
-    (pkgs.lib.makeBinPath [ pkgs.nixVersions.stable or pkgs.nix_2_4 pkgs.nixpkgs-review pkgs.nix-prefetch-git ])
+    (pkgs.lib.makeBinPath [ pkgs.nixVersions.stable or pkgs.nix_2_4 pkgs.nixpkgs-review pkgs.nix-prefetch-git pkgs.nixfmt-rfc-style ])
```

I would like to avoid haskell here because it doesn't work on some platforms
and adds quite a bit of dependencies:

```suggestion
    (pkgs.lib.makeBinPath [ pkgs.nixVersions.stable or pkgs.nix_2_4 pkgs.nixpkgs-review pkgs.nix-prefetch-git ])
```

This is also because `--format` is likely a rare use case and most people shouldn't need to download it.